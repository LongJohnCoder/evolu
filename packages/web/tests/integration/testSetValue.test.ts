import { pageDom, pageGoto } from './helpers';

beforeEach(async () => {
  await pageGoto('testSetValue');
});

test('operations', async () => {
  await page.waitFor(2000);
  await expect(await pageDom()).toMatchSnapshot();
});
