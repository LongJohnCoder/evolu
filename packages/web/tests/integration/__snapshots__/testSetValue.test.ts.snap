// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`operations 1`] = `
<div
  id="__next"
>
  <div
    autocorrect="off"
    contenteditable="true"
    data-gramm="true"
    role="textbox"
    spellcheck="false"
    tabindex="0"
  >
    <div
      class="root"
    >
      <div
        class="heading"
      >
        ·heading·
      </div>
    </div>
  </div>
  <pre
    class="jsx-2255726786"
  >
    <span
      class="jsx-4269328595"
      data-json="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'heading'},'children':['heading']}]}"
    >
      ·6 {'hasFocus':true,'selection':{'anchor':[0,0,0],'focus':[0,0,2]}}·
    </span>
    <span
      class="jsx-4269328595"
      data-json="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'paragraph'},'children':['paragraph']},{'tag':'div','props':{'className':'heading'},'children':['foo']}]}"
    >
      ·5 {'hasFocus':true,'selection':{'anchor':[0,0,0],'focus':[0,0,0]}}·
    </span>
    <span
      class="jsx-4269328595"
      data-json="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'heading'},'children':['foo']},{'tag':'div','props':{'className':'paragraph'},'children':['paragraph']}]}"
    >
      ·4 {'hasFocus':true,'selection':{'anchor':[0,0,1],'focus':[0,0,1]}}·
    </span>
    <span
      class="jsx-4269328595"
      data-json="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'heading'},'children':['heading']},{'tag':'div','props':{'className':'paragraph'},'children':['paragraph']}]}"
    >
      ·3 {'hasFocus':true,'selection':{'anchor':[0,0,0],'focus':[0,0,2]}}·
    </span>
    <span
      class="jsx-4269328595"
      data-json="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'heading'},'children':['foo']},{'tag':'div','props':{'className':'paragraph'},'children':['paragraph']}]}"
    >
      ·2 {'hasFocus':true,'selection':{'anchor':[0,0,0],'focus':[0,0,0]}}·
    </span>
    <span
      class="jsx-4269328595"
      data-json="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'heading'},'children':['heading']},{'tag':'div','props':{'className':'paragraph'},'children':['paragraph']}]}"
    >
      ·1 {'hasFocus':true,'selection':{'anchor':[0,0,0],'focus':[0,0,0]}}·
    </span>
    <span
      class="jsx-4269328595"
      data-json="{'tag':'div','props':{'className':'root'},'children':[{'tag':'div','props':{'className':'heading'},'children':['heading']},{'tag':'div','props':{'className':'paragraph'},'children':['paragraph']}]}"
    >
      ·0 {'hasFocus':true}·
    </span>
  </pre>
  <b
    id="done"
  >
    ·done·
  </b>
</div>
`;
